src/main/app-updater.ts

```diff
@@ -38,9 +40,26 @@ export function startUpdateChecking(interval = 1000 * 60 * 60 * 24): void {
 
   autoUpdater
     .on("update-available", (args: UpdateInfo) => {
+      if (autoUpdater.autoInstallOnAppQuit) {
+        // a previous auto-update loop was completed with YES+LATER, check if same version
+        if (installVersion === args.version) {
+          // same version, don't broadcast
+          return;
+        }
+      }
+
+      /**
+       * This should be always set to false here because it is the reasonable
+       * default. Namely, if a don't auto update to a version that the user
+       * didn't ask for.
+       */
+      autoUpdater.autoInstallOnAppQuit = false;
+
       try {
         const backchannel = `auto-update:${args.version}`;
 
+        installVersion = args.version;
```

Comment:
Should `installVersion` be reset in the catch block below?

Reply:
Good point it probably should be